samza-kv/src/main/scala/org/apache/samza/storage/kv/AccessLoggedStore.scala

```diff
@@ -91,6 +92,13 @@ class AccessLoggedStore[K, V](
     store.all()
   }
 
+  def iterate(from: K, to: K): KeyValueIterable[K, V] = {
+    val list : util.ArrayList[K] = new util.ArrayList[K]()
+    list.add(from)
+    list.add(to)
+    logAccess(DBOperation.ITERATE, serializeKeys(list), store.iterate(from, to))
```

Comment:
why do you need this list? is it the name of the iterator?

Reply:
Oh, I believe this is for logging every access of the store to a stream.